web-client/integration-tests-public/journey/unauthedUserAttemptsToViewStrickenDocumentUnsuccessfully.js

```diff
@@ -0,0 +1,11 @@
+export const unauthedUserAttemptsToViewStrickenDocumentUnsuccessfully = test => {
+  return it('View stricken document unsuccessfully', async () => {
+    await expect(
+      test.runSequence('openCaseDocumentDownloadUrlSequence', {
+        docketEntryId: test.docketEntryId,
+        docketNumber: test.docketNumber,
+        isPublic: true,
```

Comment:
It's stricken but it's public? Is that right?

Reply:
Never mind, I think this is just saying the searcher is not an authenticated user. 👍